pkg/auth/context.go

```diff
@@ -17,50 +18,47 @@ const (
 	// FilterByOrganisation is used to determine whether resources are filtered by a user's organisation or as an individual owner
 	contextFilterByOrganisation contextKey = "filter-by-organisation"
 	contextIsAdmin              contextKey = "is_admin"
+)
 
+var (
 	// ocm token claim keys
-	ocmUsernameKey string = "username"
-	ocmOrgIdKey    string = "org_id"
-	isOrgAdmin     string = "is_org_admin" // same key used in mas-sso tokens
+	tenantUsernameClaim string = "username"
+	tenantIdClaim       string = "org_id"
+	tenantOrgAdminClaim string = "is_org_admin" // same key used in mas-sso tokens
 
 	// sso.redhat.com token claim keys
-	ssoRHUsernameKey  string = "preferred_username" // same key used in mas-sso tokens
-	ssoRhAccountIdKey string = "account_id"
+	alternateTenantUsernameClaim string = "preferred_username" // same key used in mas-sso tokens
+	tenantUserIdClaim            string = "account_id"
 
 	// mas-sso token claim keys
 	// NOTE: This should be removed once we migrate to sso.redhat.com as it will no longer be needed (TODO: to be removed as part of MGDSTRM-6159)
-	masSsoOrgIdKey = "rh-org-id"
+	alternateTenantIdClaim = "rh-org-id"
 )
 
 func GetUsernameFromClaims(claims jwt.MapClaims) string {
-	if claims[ocmUsernameKey] != nil {
-		return claims[ocmUsernameKey].(string)
-	}
-
-	if claims[ssoRHUsernameKey] != nil {
-		return claims[ssoRHUsernameKey].(string)
+	if idx, val := arrays.FindFirst(func(x interface{}) bool { return x != nil }, claims[tenantUsernameClaim], claims[alternateTenantUsernameClaim]); idx != -1 {
+		return val.(string)
 	}
-
 	return ""
 }
 
 func GetAccountIdFromClaims(claims jwt.MapClaims) string {
-	if claims[ssoRhAccountIdKey] != nil {
-		return claims[ssoRhAccountIdKey].(string)
+	if claims[tenantUserIdClaim] != nil {
+		return claims[tenantUserIdClaim].(string)
 	}
 	return ""
 }
 
 func GetOrgIdFromClaims(claims jwt.MapClaims) string {
-	if claims[ocmOrgIdKey] != nil {
-		if orgId, ok := claims[ocmOrgIdKey].(string); ok {
+	if claims[tenantIdClaim] != nil {
+		if orgId, ok := claims[tenantIdClaim].(string); ok {
```

Comment:
I think this check is useless: if `claims[tenantIdClaim] != nil` in the line above, then `ok` can't be anything else than true.

Reply:
I think the `ok`, here is for the cast to string to check if the cast was a success or not.

Reply:
My bad, you are right.